### PR TITLE
Test config

### PR DESCRIPTION
**Pull request by markjmeier**
Reworked the test configuration options article to include more samples from https://docs.k6.io/docs/options that are related to cloud execution.

### CODE REVIEW

**Comment by markjmeier** [comment]
@robingustafsson and @na-- Can one or both of you take a look at this article and let me know what you think.  There are some missing examples, especially towards the end as they weren't available in the k6 docs.

File: jekyll/_v4/test-scripting/test-configuration-options.md [hunk 1]
```diff
@@ -1,89 +1,141 @@
...
 order: 3
 ---
 
 ***
 
-<h1>Purpose</h1>
+<h1>Background</h1>
 
-This article documents the most common configuration options and how to change them in your k6 test.
+Options allow you to configure how k6 will behave during test execution.  For most options, you can specify the specific option in four different ways:
```
Comment (robingustafsson) [inline jekyll/_v4/test-scripting/test-configuration-options.md]: You double-spacer 😄

File: jekyll/_v4/test-scripting/test-configuration-options.md [hunk 2]
```diff
@@ -1,89 +1,141 @@
...
+```
 
-**NOTE:** If `vus` are specified as written above and dynamic stages are also used as described in the following section. Tests will **START** at the constant VU level specified.
+**Select the options you wish to learn more about from the following table:**
+- TOC
+{:toc}
+
+## duration
+
+**What it is:** A string specifying the total duration a test run should be run for. During this time each Virtual User will execute the script(default function) in a loop.
```
Comment (robingustafsson) [inline jekyll/_v4/test-scripting/test-configuration-options.md]: Missing space after "script" before "("?

File: jekyll/_v4/test-scripting/test-configuration-options.md [hunk 3]
```diff
@@ -1,89 +1,141 @@
...
+            name: "My test name"
+        }
+    }
+};
+{% endhighlight %}
 
-## Sending results to a specific project
+### project
 
 <div class="callout callout-warning" role="alert">
```
Comment (robingustafsson) [inline jekyll/_v4/test-scripting/test-configuration-options.md]: This is not true anymore (and by some accounts was never true). We should remove this alert. I've removed it in other places in the KB and in the k6 docs.

Comment (robingustafsson) [reply jekyll/_v4/test-scripting/test-configuration-options.md]: Also `K6_CLOUD_PROJECT_ID ` is currently broken in all official releases of k6, see note I added here: https://docs.k6.io/v1.0/docs/cloud-execution#section-running-tests-under-a-different-project-than-your-default-one

File: jekyll/_v4/test-scripting/test-configuration-options.md [hunk 4]
```diff
@@ -165,6 +468,164 @@ export let options = {
...
+
+**Example:**
+
+{% highlight js linenos %}
+code here
+{% endhighlight %}
+
+***
+
+## config.json
```
Comment (robingustafsson) [inline jekyll/_v4/test-scripting/test-configuration-options.md]: Something missing here?

File: jekyll/_v4/test-scripting/test-configuration-options.md (hunk 1 above)
Comment (markjmeier) [reply jekyll/_v4/test-scripting/test-configuration-options.md]: 🤦‍♂️

**Comment by markjmeier** [comment]
@robingustafsson Updated - I ultimately don't want to maintain this list in two spots, but I have seen some users ask about these types of things and having the article in the kb makes it searchable.  I prefer to redirect users to the k6 docs, but also need it to be easily accessible to users who may not know those exist.

File: jekyll/_v4/test-scripting/test-configuration-options.md [hunk 5]
```diff
@@ -165,6 +463,167 @@ export let options = {
...
+
+Specify the config file in JSON format to read the options values. If the config file is not specified, k6 will look up for `config.json` in the `loadimpact/k6` directory inside the regular directory for configuration files on the operational system.
```
Comment (robingustafsson) [inline jekyll/_v4/test-scripting/test-configuration-options.md]: "will look up for" sounds strange no?

also "operational system" should be "operating system"

Comment (markjmeier) [reply jekyll/_v4/test-scripting/test-configuration-options.md]: Yeah took the shortcut and copied it directly from https://docs.k6.io/docs/options Guess I'll submit an edit there too.

**Comment by markjmeier** [comment]
Merged.

**Pull request merged**